core/src/apps/management/reset_device/layout.py

```diff
@@ -245,76 +289,67 @@ def _get_mnemonic_page(words: list):
 # TODO: smaller font or tighter rows to fit more text in
 # TODO: icons in checklist
```

Comment:
Let's remove these TODOs for now. There should be an issue, if it really bothers someone :).

Reply:
done